internal/locale/locales/en-us.yaml

```diff
@@ -2043,3 +2043,5 @@ runtime_limit_reached_action:
   other: Upgrade
 pjfile_deprecation_entry:
   other: " - '[ACTIONABLE]{{.V0}}[/RESET]' located at byte [ACTIONABLE]{{.V1}}[/RESET]"
+err_country_blocked:
+  other: Your country is blocked
```

Comment:
Oof. That's a bit harsh. We might want to soften the language a bit, but I'm not sure how.

Reply:
Agreed it is a bit rough, but it is from the ACs of the story. @Naatan do we want to consider a more friendly error message?

Reply:
@rawktron ^

Reply:
"This service is not available in your region."